src/Routing/Route.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of sensiolabs-de/storyblok-bundle.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Bundle\Routing;
+
+enum Route
+{
+    public const string STORYBLOK_WEBHOOK = 'storyblok_webhook';
```

Comment:
```suggestion
    public const string ROUTE_WEBHOOK = 'storyblok_webhook';
```
?

Or only WEBHOOK ?

We are in a Storyblok context already

Reply:
Not a big fan of it. That would require deprecating the route name etc. I think introducing the Route class does not belong in this PR i will remove it from this scope

Reply:
i did that here:

- #27